frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -344,8 +356,29 @@ export class EnrolledItemCard extends React.Component<
           )}
 
           <div className="col-12 col-md px-3 py-3 py-md-0 box">
-            <div className="d-flex justify-content-between align-content-start flex-nowrap w-100 enrollment-mode-container">
-              <h2 className="my-0 mr-3">{title}</h2>
+            <div className="d-flex justify-content-between align-content-start flex-nowrap w-100">
+              <div className="d-flex flex-column">
+                <div className="align-content-start d-flex enrollment-mode-container">
+                  {hasPassed ? (
+                    <span className="badge badge-enrolled-passed mr-2">
+                      <img src="/static/images/done.svg" alt="Check" /> Course
+                      passed
+                    </span>
+                  ) : null}
+                  {enrollment.enrollment_mode === "verified" ? (
+                    <span className="badge badge-enrolled-verified mr-2">
+                      Enrolled in certificate track
+                    </span>
+                  ) : null}
+                  {startDateDescription !== null ? (
```

Comment:
Does `startDateDescription` always have a defined value as long as a start_date is set for the course_run?